nix/modules/kld/default.nix

```
@@ -70,6 +70,13 @@ in
       '';
     };
 
+    mnemonicPath = lib.mkOption {
+      type = lib.types.path;
```

You don't need the second option. You can just do:

```suggestion
      type = lib.types.nullOr lib.types.path;
      default = null;
```